### ISSUE TITLE
Stats: Improve `model.md`

### ISSUE THREAD

**Issue by dp-rice** [naobservatory/p2ra#64]
1. Focus on the conceptual/scientific description rather than jumping right to Stan code.
2. Separate the model of prevalence from the rest of the model conditional on prevalence.
3. Add some figures: priors, posteriors, model schematic

**Comment by dp-rice** [comment]
After #106

**Comment by dp-rice** [comment]
Unclear whether this is worth doing once the blog post is up (and eventually the preprint). On one hand, it can be good to have documentation in the repo. On the other hand, it's annoying to have multiple redundant descriptions of the same thing to maintain.
@jeffkaufman what do you think?

**Comment by jeffkaufman** [comment]
I don't think this is worth it, no.  We can just point people to the online version. If we have additional details we want to document, technical ones that are too in-the-weeds for the online version, though, the repo is a good place?
